src/v/cloud_storage/tests/CMakeLists.txt

```diff
@@ -48,6 +48,7 @@ rp_test(
   DEFINITIONS BOOST_TEST_DYN_LINK
   LIBRARIES v::seastar_testing_main Boost::unit_test_framework v::cloud_storage v::storage_test_utils
   LABELS cloud_storage
+  ARGS "-- -c 8"
```

Comment:
nit: it would be nice to provide tests with a fixed memory allocation too. I always run them with `-m1G` as they behave nicer when killed.

Reply:
That is a great point, I'm trying it now + will add a commit if all good.